.travis/run-parts

```diff
@@ -13,9 +13,25 @@ if [[ "${dir}" = "" || ! -d "${dir}" ]]; then
     exit 1
 fi
 
-for f in $(echo ${dir%/}/*); do
-	[ -d $f ] && continue
-	[ ! -x $f ] && continue
-	echo "+$f"
-	$f
-done
\ No newline at end of file
+fold_start() {
+    echo -e "travis_fold:start:$1\033[33;1m$2\033[0m"
+}
+
+fold_end() {
+    echo -e "\ntravis_fold:end:$1\r"
+}
+
+for tagset in '' 'bounds' 'noasm' 'appengine' ;
+do
+    export TAGS=$tagset
```

Comment:
This needs to include the `-tags=` prefix if `$tagset` is set.

Reply:
Ah, right, still trying to get everything to work nicely :). Thanks.